taca/server_status/server_status.py

```diff
@@ -0,0 +1,79 @@
+import subprocess
+import json
+import gspread
+from oauth2client.client import SignedJwtAssertionCredentials as GCredentials
+from taca.utils.config import CONFIG
+
+def get_disk_space():
+	result = {}
+	config = CONFIG['server_status']
+	servers = config.get('servers', [])
+	for server_url in servers.keys():
+		# get path of disk
+		path = servers[server_url]
+
+		# get command
+		command = config['command'] + ' ' + path
+
+		# if localhost, don't connect to ssh
+		if server_url == "localhost":
+			proc = subprocess.Popen(command.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+		else:
+			# connect via ssh to server and execute the command
+			proc = subprocess.Popen(['ssh', '-t', '%s@%s' %(config['user'], server_url), command],
+				stdout = subprocess.PIPE,
+				stderr = subprocess.PIPE)
+		# read output
+		output = proc.stdout.read()
+		# parse output
+		output = __parse_output(output)
+
+		try:
+			# remove % symbol and convert string to int
+			used_space = int(output.strip().replace('%', ''))
+			# how many percent available
+			available_space = 100 - used_space
+			result[server_url] = str(available_space) + '%'
+		except:
+			# sometimes it fails for whatever reason as Popen returns not what it is supposed to
```

Comment:
not a big fun of this.... , print to the log a warning and then continue.... however would be better to fail as we the risk is that we update with wrong values. 
Test is on Preproc1, 2, and B5. If it works then it is fine and this line can became an exception to be thrown